v0.0.1/admin-schema.json

```diff
@@ -0,0 +1,633 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/admin-schema.json",
```

Comment:
I noticed that the other two files followed a different indentation convention, with two spaces instead of four. Consider updating this file to also use two spaces for indentation?

Reply:
I've now harmonised indentation in both files. However, I made the executive decision that 4 spaces makes the document much easier to read and work with as the nesting is much more obvious so I've converted both files to 4 spaces 😁